cmd/process-agent/command/main_common.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/DataDog/datadog-agent/comp/core/sysprobeconfig"
 	"github.com/DataDog/datadog-agent/comp/core/sysprobeconfig/sysprobeconfigimpl"
 	"github.com/DataDog/datadog-agent/comp/core/tagger"
```

Comment:
Do we still need this import since it is replace by `taggerimpl`

Reply:
We reference the tagger package in multiple places:
https://github.com/DataDog/datadog-agent/blob/d4b130e60789a8e231e3228858727ee116bd0c7c/cmd/process-agent/command/main_common.go#L176

https://github.com/DataDog/datadog-agent/blob/d4b130e60789a8e231e3228858727ee116bd0c7c/cmd/process-agent/command/main_common.go#L203

Reply:
okay didn't saw thx